requirements.txt

```diff
@@ -1,6 +1,33 @@
-matplotlib==3.0.2
-PySide==1.2.4
-# PySide2==5.12.0
-Shiboken==1.2.2
-six==1.12.0
-Markdown==3.2.2
+area==1.1.1
+bzr_pqm==1.4.0
+cog==0.6.1
+ConfigParser==5.3.0
+defusedxml==0.7.1
+GitPython==3.1.31
+ifcopenshell==0.7.0.230318
+kdtree==0.16
+ladybug==0.0.2
+matplotlib==3.6.3
+numpy==1.24.2
+ocl==0.12
+opencamlib==2023.1.11
+packaging==23.0
+Pivy==0.6.8
+ply==3.11
+ptvsd==4.3.2
+pyNastran==1.3.4
+PyQt4==4.11.4
+PyQt5==5.15.9
```

Comment:
this can't be correct, we dropped support for qt4 a long time ago and IIRC we are not using pyqt at all from a long time also.

Reply:
Looks like there's some files that still use it https://github.com/FreeCAD/FreeCAD/search?q=pyqt4
Should those files be fixed?

Reply:
Yeah, there's no way those actually work now. File an issue and we'll track down who knows something about them.

Reply:
We also don't use PyQt5, we use PySide.